shared/selene/api/endpoints/account.py

```diff
@@ -373,11 +374,28 @@ def _update_open_dataset_agreement(self, opt_in: bool):
             self.account_activity_repository.increment_open_dataset_deleted()
 
     def delete(self):
+        """Process a HTTP DELETE request for an account."""
         self._authenticate()
         self.account_repository.remove(self.account)
         self.account_activity_repository.increment_accounts_deleted()
-        membership = self.account.membership
-        if membership is not None:
-            cancel_stripe_subscription(membership.payment_id)
+        if self.account.membership is not None:
+            cancel_stripe_subscription(self.account.membership.payment_id)
+        self._change_wake_word_file_status()
```

Comment:
so status is the same for members and non menmbers?

Reply:
membership is independent of opting in or out of the data agreement